debezium-core/src/main/java/io/debezium/relational/TableId.java

```diff
@@ -37,6 +37,12 @@ public static TableId parse(String str) {
     public static TableId parse(String str, boolean useCatalogBeforeSchema) {
         String[] parts = str.split("[\\" + '.' + "]");
         if ( parts.length < 0 ) return null;
+        for (int i = 0; i < parts.length; i++) {
+            String part = parts[i];
+            if ( part.startsWith("\"") && part.endsWith("\"") ) {
+                parts[i] = part.substring(1, part.length()-1);
+            }
+        }
```

Comment:
Rather than it be a String are you ok with it taking an interface that allows custom transformation?

e.g.

    public interface TableIdTransformer {
        String fromSqlQuoted(String part);   
        String toSqlQuoted(String part);
    }

If provided then I can use that in both the parse and quotedTableId methods?